.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,20 @@
-#### What does this PR do?
-#### Any background context you want to provide?
-#### What ticket does this PR close?
-Connected to [relevant GitHub issues, eg #76]
-#### Where should the reviewer start?
-#### How should this be manually tested?
-#### Screenshots (if appropriate)
-#### Has the Version and Changelog been updated?
-#### Questions:
-> Does this work have automated integration and unit tests?
+### What does this PR do?
+- _What's changed? Why were these changes made?_
+- _How should the reviewer approach this PR, especially if manual tests are required?_
+- _Are there relevant screenshots you can add to the PR description?_
 
-> Can we make a blog post, video, or animated GIF of this?
+### What ticket does this PR close?
+Connected to #[relevant GitHub issues, eg 76]
 
-> Has this change been documented (Readme, docs, etc.)?
+### Checklists
 
-> Does the knowledge base need an update?
+#### Change log
+- [ ] The CHANGELOG has been updated, or
+- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update
+
+#### Test coverage
+- [ ] This PR includes new unit or integration tests to go with the code changes
+
+#### Follow-on issues
+- [ ] Follow-up issue(s) have been logged to update documentation or related projects, or
```

Comment:
What do you think about having the PR include links to followup tickets? I wonder if it'll help insure these issues are actually filed.
```suggestion
- [ ] Follow-up issue(s) have been logged (and links included below) to update documentation or related projects, or
```